crates/holochain-conductor-runtime/src/runtime.rs

```diff
@@ -181,6 +175,33 @@ impl Runtime {
         }
     }
 
+    /// Full process to setup an app
+    ///
+    /// Check if app is installed, if not install it, then optionally enable it.
+    /// Then ensure there is an app websocket and authentication for it.
+    pub async fn setup_app(
```

Comment:
I find this behavior a bit unexpected. It has an install app payload, but if the app is already installed, the function doesn't error and just enables it, but only if the enable option is true. I would expect the function to return an error if the app is already installed, or at least log that.

Reply:
Hmm, my rationale is to have a single function that I can call in the webview initialization script that does the entire setup process necessary to inject the `__HC_LAUNCHER_ENV__`. If the app is already installed and we already have a websocket, then it is a no-op.

This lets us drop several other commands from the tauri plugin that don't need to be exposed outside of this single use case: `connect`, `isAppInstalled`, `installApp`, `ensureAppWebsocket`.

Sure I can log when the app is already installed: https://github.com/holochain/android-service-runtime/pull/87/commits/b71b52b2bc1f9bdc5f036dd1b3b813764ecfbe47

Reply:
expanded the doc comment: https://github.com/holochain/android-service-runtime/pull/87/commits/4ccda728b7793be19adf4c4cd8d50baaf39b71da